src/pages/projects/[project]/components/ProjectContent/hook/useQuery.ts

```diff
@@ -28,11 +30,22 @@ const useQuery = () => {
     }
   }
 
+  async function loadProjectMeasuresHistory() {
+    if (query?.project) {
+      try {
+        const result = await projectQuery.getProjectMeasuresHistory('1', query?.project as string);
+        return result.data;
```

Comment:
WDT?

`const [projectMeasuresHistory, setProjectMeasuresHistory] = useState();`

```suggestion
        setProjectMeasuresHistory(result.data);
```

`return { project, projectMeasuresHistory };`

Reply:
Hmmm... Eu iria implementar dessa maneira. Mas colocando a chamada da função `loadProjectMeasuresHistory` no `useEffect`, acabaria realizando requisições desnecessária em páginas que utilizam o `useQuery`, mas não utilizam esse estado `projectMeasuresHistory`.  

Para ser sincero, até prefiro da maneira como sugeriu 😅. Se concordar que esse não um problema tão relevante, rola fazer essa atualização